File: fix_commas.js
const fs = require('fs');
const filePath = '/mnt/c/Users/User/plizio-repo/lib/visualLab/data/poiExtraRo2.ts';
let content = fs.readFileSync(filePath, 'utf8');

// The issue is:
// facts: {
//   ...
// }
// descriptionAdvanced: {

// It should be:
// facts: {
//   ...
// },
// descriptionAdvanced: {

content = content.replace(/\n\s*\}\n\s*descriptionAdvanced:/g, '\n    },\n    descriptionAdvanced:');

fs.writeFileSync(filePath, content, 'utf8');
console.log('Fixed missing commas');

File: fix_commas.py
import re

def process_file():
    with open('lib/visualLab/data/faq/deFaq.ts', 'r', encoding='utf-8') as f:
        content = f.read()

    # Find occurrences of `    ]\n    en: [` and replace with `    ],\n    en: [`
    new_content = re.sub(r'(\s+)\](\s+)en: \[', r'\1],\2en: [', content)

    with open('lib/visualLab/data/faq/deFaq.ts', 'w', encoding='utf-8') as f:
        f.write(new_content)

process_file()
print("Fixed missing commas.")
